src/cc65/pragma.c

```diff
@@ -125,7 +125,7 @@ static const struct Pragma {
     { "static-locals",          PRAGMA_STATIC_LOCALS      },
     { "staticlocals",           PRAGMA_STATICLOCALS       },      /* obsolete */
     { "warn",                   PRAGMA_WARN               },
-    { "wrapped-call",		PRAGMA_WRAPPED_CALL	  },
+    { "wrapped-call",           PRAGMA_WRAPPED_CALL	  },
```

Comment:
Good that we follow up on this: Apart from Makefiles there should be no tab (ascii 9) char in any cc65 source file! See https://github.com/cc65/wiki/wiki/C-and-Assembly-sources-style-guide